src/robot/model/body.py

```diff
@@ -33,7 +32,7 @@
     from .testsuite import TestSuite
 
 
-BodyItemParent = Union['TestSuite', 'TestCase', 'UserKeyword', 'For', 'ForIteration',
+BodyItemParent = Union['TestSuite', 'TestCase', 'Keyword', 'For', 'ForIteration',
```

Comment:
This change isn't totally correct. `Keyword` can be a parent only in the result model. `UserKeyword` only exists in the running model so having it here in the base may not be a good idea, though.